styles/styles.css

```diff
@@ -0,0 +1,582 @@
+ul,
+li {
+  list-style: none;
+  margin: 0;
+  padding: 0;
+}
+
+body {
+  font-family: 'Work Sans', sans-serif;
+  font-size: .8rem;
+  line-height: 1.4rem;
+}
+
+h1 {
+  font-family: 'Neuton', sans-serif;
+  color: #093c60;
+  font-size: 2rem;
+}
+
+h2 {
+  font-family: 'Neuton', sans-serif;
+  color: #093c60;
+  font-size: 1.4rem;
+  margin: 0;
+}
+
+h3 {
+  font-family: 'Neuton', sans-serif;
+  color: #093c60;
+  font-size: 1.2rem;
+  margin: 0;
+}
+
+h4,
+h5,
+h6 {
+  font-family: 'Neuton', sans-serif;
+  color: #093c60;
+  margin: 0;
+}
+
+
+/* skip link */
+
+.hidden {
+  position: relative;
+  left: -10000px;
+  top: auto;
+}
+
+.hidden:focus {
+  left: 0;
+}
+
+
+/* header */
+
+header {
+  background-image: url("../images/header-image.jpg");
+  background-size: cover;
+  padding: 2rem 6rem 6rem;
+}
+
+.header-headline {
+  color: #81b741;
+  font-style: italic;
+  line-height: .2rem;
+  margin: 0;
+}
+
+.flex-header {
+  display: flex;
+  height: 17rem;
+  flex-wrap: wrap;
+}
+
+.flex-header-1 {
+  flex: 1;
+  align-items: flex-start;
+}
+
+.flex-header-1 img {
+  width: 18.5rem;
+}
+
+.flex-header-2 {
+  margin-left: auto;
+}
+
+.flex-cart {
+  display: flex;
+  height: 3rem;
+  width: 100px;
+  flex-wrap: nowrap;
+}
+
+.flex-cart img {
+  width: 20px;
+}
+
+.flex-icon {
+  flex: 25px;
+  padding: 0px 15px 10px;
+}
+
+.flex-price {
+  flex: 30px;
+  color: white;
+}
+
+.green-button {
+  margin: 0;
+  color: white;
+  font-weight: 600;
+  text-align: center;
+  background-color: #81b741;
+  height: 2rem;
+  width: 100%;
+  padding-top: .3rem;
+}
+
+
+/* top navigation */
+
+main nav:first-of-type {
+  background: #ececec;
+  box-shadow: 0rem .2rem .7rem #c9c9c9;
+  padding-left: 6rem;
+}
+
+nav {
+  font-weight: 600;
+}
+
+.flex-nav {
+  display: flex;
+  width: 250px;
+  height: 3rem;
+  flex-wrap: nowrap;
+  padding: .8rem 0rem 1.5rem;
+}
+
+.flex-nav-home {
+  flex: 1.5rem;
+}
+
+.flex-nav-dogs {
+  flex: 1.5rem;
+}
+
+.flex-nav-blog {
+  flex: 3rem;
+}
+
+/* add slash after nav menu items */
+
+.flex-nav-home a::after {
```

Comment:
You could just one one class and call the same class after the elements that need it